buildSrc/subprojects/parser/src/test/java/org/apache/calcite/test/DialectGenerateTest.java

```diff
@@ -444,11 +444,11 @@ private void assertFileProcessed(String testName, boolean specifyFilePath) {
     extractedData.nonReservedKeywords.add(new Keyword("foo"));
     dialectGenerate.unparseNonReservedKeywords(extractedData);
     assertEquals(2, extractedData.functions.size());
-    assertTrue(extractedData.functions.containsKey("NonReservedKeyword"));
+    assertTrue(extractedData.functions.containsKey("NonReservedKeyWord"));
     assertTrue(extractedData.functions.containsKey("NonReservedKeyword0of1"));
-    String actual = extractedData.functions.get("NonReservedKeyword");
+    String actual = extractedData.functions.get("NonReservedKeyWord");
```

Comment:
nit: Could this be kept as "Keyword" since some variable names and autogenerated methods still have that format? Or the other way around to keep it constant

Reply:
I'll try changing it back, I had some build issues before for some reason.

Reply:
I ended up changing them to all have the capital W, it was more straightforward.